bundle/Enums/VisibilityAction.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Enums;
+
+enum VisibilityAction: string
```

Comment:
Let's rename this to `VisibilityUpdateResult`.

Reply:
fixed in 00985a094e795f79104e4b1c94129519cf864bbb